.circleci/config.yml

```diff
@@ -123,10 +123,19 @@ workflows:
   version: 2
   build-test-deploy:
     jobs:
-      - checkout_code
+      - checkout_code:
+          filters:
```

Comment:
I wonder if this is necessary. The one that matters (and the one that should drive things) is `build_test_and_deploy` below. If filters says that one should run, does it trigger running `checkout_code` automatically thanks to the `requires:` directive?

Reply:
"2. For a tag push unaffected by any filters, CircleCI skips the job.
...
Item two above means that a job must have a filters tags section to run as a part of a tag push and _all its transitively dependent jobs_ must also have a filters tags section."
- https://circleci.com/docs/2.0/workflows/#git-tag-job-execution